src/mixins/bcol-mixin.ts

```diff
@@ -0,0 +1,30 @@
+import { Component, Vue } from 'vue-property-decorator'
+import axios from '@/axios-auth'
+
+@Component({})
+export default class BcolMixin extends Vue {
+  // readonly payApiUrl: string
+
+  async getErrorObj (errCode) {
```

Comment:
Please add return types to all methods in this mixin.

Reply:
I considered this, but I didn't see a test file for paymenterrordialog. Will do

Reply:
2 out of 3 are done. Please also do `get payApi ()`.

Reply:
Do you use VS plugins Vetur and Vue Peek? Those might not identify that a return type is missing, but they should identify errors if you specify one type and use it differently.

Reply:
I do not use either of those :/ I will install them...and whoops I guess I missed this one